contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -717,3 +717,29 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+/*
+* CONSTRAINT TABLE USAGE
+*/
+CREATE OR REPLACE VIEW information_schema_tsql.constraint_table_usage
+AS SELECT CAST(sys.db_name() AS sys.nvarchar(128)) AS "TABLE_CATALOG",
+    CAST((select orig_name from sys.babelfish_namespace_ext where nr.nspname = nspname) AS sys.nvarchar(128)) AS "TABLE_SCHEMA",
+    CAST(r.relname AS sys.sysname) AS "TABLE_NAME",
+    CAST(sys.db_name() AS sys.nvarchar(128)) AS "CONSTRAINT_CATALOG",
```

Comment:
instead of directly printing sys.db_name() - we should add that as condition to filter out the query result.
i.e. babelfish_namespace_ext.dbid = sys.db_id()
Please refer existing information_schema_tsql views for reference.